lib/lib_rss.php

```diff
@@ -631,11 +631,7 @@ function invalidateHttpCache(string $username = ''): bool {
 		Minz_Session::_param('touch', uTimeString());
 		$username = Minz_User::name() ?? Minz_User::INTERNAL_USER;
 	}
-	$ok = @touch(DATA_PATH . '/users/' . $username . '/' . LOG_FILENAME);
-	//if (!$ok) {
-		//TODO: Display notification error on front-end
```

Comment:
The error notification is taken care of with the refactor?

Reply:
Yes, the Boolean return value is tested by the callers when it makes sense, for instance:
https://github.com/FreshRSS/FreshRSS/blob/fe423e19ee326e4f8d87f23ec4eb10635b76ee60/app/actualize_script.php#L108-L112